src/views/product-integration/component-view/components/searchable-variable-group-list/index.tsx

```diff
@@ -80,7 +80,10 @@ export default function SearchableVariableGroupList({
 			<p className={s.results}>
 				{numMatches} {numMatches === 1 ? 'Result' : 'Results'}
 			</p>
-			<VariableGroupList variables={matchesWithAncestors} />
+			<VariableGroupList
+				groupName={groupName}
```

Comment:
Passing `groupName` as it's used to construct the anchor links slugs for each variable (to ensure anchor links slugs are unique across variable groups).